ui/admin/tests/e2e/global-setup.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-undef */
```

Comment:
As per my understanding, eventually we will need, but I will say do not worry just now because in the UI repository we did not applied them to any files.